### PR TITLE
Small `pgdb` cleanup

### PR DESCRIPTION
**Pull request by AlekSi**
Small cleanup before bigger changes.

## Readiness checklist

* [x] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [x] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [x] I checked comments rendering with `task godocs`.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/handlers/pg/pgdb/databases.go [hunk 1]
```diff
@@ -133,3 +133,26 @@ func CreateDatabaseIfNotExists(ctx context.Context, querier pgxtype.Querier, db
 		return lazyerrors.Error(err)
 	}
 }
+
+// DropDatabase drops FerretDB database.
+//
+// It returns ErrTableNotExist if schema does not exist.
```
Comment (rumyantseva) [inline internal/handlers/pg/pgdb/databases.go:139]:
```suggestion
// It returns ErrSchemaNotExist if schema does not exist.
```

**Review by rumyantseva — CHANGES_REQUESTED** [review]
One piece of doc is not correct (see my suggestion) and linters fail, but overall the changes look reasonable.

File: internal/handlers/pg/pgdb/collections.go [hunk 2]
```diff
@@ -160,6 +160,40 @@ func CreateCollection(ctx context.Context, querier pgxtype.Querier, db, collecti
...
+	}
+
+	// Table (or even schema) does not exist. Try to create it,
+	// but keep in mind that it can be created in concurrent connection.
+
+	if err := CreateDatabase(ctx, querier, db); err != nil && !errors.Is(err, ErrAlreadyExist) {
+		return false, lazyerrors.Error(err)
+	}
+
+	// TODO use a transaction instead of pgPool: https://github.com/FerretDB/FerretDB/issues/866
```
Comment (rumyantseva) [inline internal/handlers/pg/pgdb/collections.go:185]: Is this comment still actual? It seems that we already use querier here.

Comment (AlekSi) [reply internal/handlers/pg/pgdb/collections.go:185]: But we don't use a transation

Comment (rumyantseva) [reply internal/handlers/pg/pgdb/collections.go:185]: Ok, got it (now I see how transaction is used in update).

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**